modules/migration/comment.go

```diff
@@ -7,6 +7,13 @@ package migration
 
 import "time"
 
+// Commentable can be commented upon
+type Commentable interface {
+	GetLocalID() int64
```

Comment:
The meaning of `ID` is quite confusing.

In Gitea, `ID` means the primary key of a model, for example, IssueID is `issue`.`id`

The `number` of an issue is called `index` in Gitea.

Should we clarify these concepts? `GetLocalIssueIndex` is more clear if I understand correctly.